src/FocusableView/FocusableView.tsx

```diff
@@ -133,7 +121,7 @@ export class FocusableView extends React.Component<FocusableView.Props> {
     return React.createElement(
       component as any,
       locals,
-      t.Function.is(children) ? children(focused) : children
+      isFunction(children) ? children(focused) : children
```

Comment:
why not `typeof children === "function"` as you did for strings and numbers?
That's the same exact thing lodash's `isFunction` does.

Reply:
makes sense, I'll do it